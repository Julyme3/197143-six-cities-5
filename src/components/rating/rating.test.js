import React from "react";
import renderer from "react-test-renderer";
import Rating from "./rating";


test(`Rating render`, () => {
  const tree = renderer
    .create(
        <Rating
          rating={3}
          className="className"
        />
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
