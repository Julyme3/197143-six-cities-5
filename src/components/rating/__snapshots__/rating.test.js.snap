// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rating render 1`] = `
<div
  className="className rating__stars"
>
  <span
    style={
      Object {
        "width": "60%",
      }
    }
  />
  <span
    className="visually-hidden"
  >
    Rating
  </span>
</div>
`;
